api/v1/azureappconfigurationprovider_types.go

```diff
@@ -79,8 +79,9 @@ type AzureAppConfigurationFeatureFlagOptions struct {
 
 // KeyLabelSelector defines the filters when fetching the data from Azure AppConfiguration
 type Selector struct {
-	KeyFilter   string  `json:"keyFilter"`
-	LabelFilter *string `json:"labelFilter,omitempty"`
+	KeyFilter    *string `json:"keyFilter,omitempty"`
+	LabelFilter  *string `json:"labelFilter,omitempty"`
+	SnapshotName *string `json:"snapshotName,omitempty"`
```

Comment:
why is this not named `xxxFilter`? sounds inconsistent

Reply:
This is discussed in the design, `snapshotFilter` sounds not reasonable.